ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -494,9 +575,18 @@ private void initializeDisksAndReplicasOnNode(AmbryDataNode datanode, InstanceCo
             }
             ensurePartitionAbsenceOnNodeAndValidateCapacity(mappedPartition, datanode, replicaCapacity);
             // Create replica associated with this node.
+            boolean isSealed;
+            if (clusterMapConfig.clusterMapEnablePartitionOverride && partitionOverrideInfoMap.containsKey(
+                partitionName)) {
```

Comment:
just want to make sure that partitionName equals the result of toPathString in this case

Reply:
`"2443:412316860416:max-replicas-all-datacenters,2533:412316860416:max-replicas-all-datacenters,` is an example in InstanceConfig. The `partitionName` here is `2443`, `2553` etc, which should equal to the result of `toPathString`.